docs-source/content/userguide/cicd/tools.md

```diff
@@ -22,7 +22,7 @@ interesting.  A good way to get a jumpstart on a model is to use the
 `discoverDomain` tool in WDT which builds a model based on an existing domain.
 
 {{% notice note %}}
-A Model in Image domain takes complete advantage of the WDT tool. It let's
+A Model in Image domain completely takes advantage of the WDT tool. It let's
```

Comment:
awkward ;p

Reply:
lol

Reply:
I agree (Tom's original wording, reworded). I will remove the word "completely", completely. Does that work for you?

Reply:
Works for me.

Reply:
completely